Sources/ComposableArchitecture/Store.swift

```diff
@@ -131,7 +131,7 @@ public final class Store<State, Action> {
     private let reducer: (inout State, Action) -> EffectTask<Action>
     fileprivate var scope: AnyStoreScope?
   #endif
-  var state: CurrentValueSubject<State, Never>
+  @_spi(Internals) public var state: CurrentValueSubject<State, Never>
```

Comment:
added this so that I could more fully test the `Store` in complete isolation (no need to interface with `ViewStore`). I just wanted to be 100% we were asserting on the full behavior inside the store without outside factors.